app/src/main/java/com/ybsystem/tweethub/activities/SearchActivity.java

```diff
@@ -0,0 +1,221 @@
+        // Set search pager
+        mSearchPager = findViewById(R.id.pager_common);
+        mSearchPager.setOffscreenPageLimit(3);
+        mSearchPagerAdapter = new SearchPagerAdapter(getSupportFragmentManager(), mSearchWord);
+        mSearchPager.setAdapter(mSearchPagerAdapter);
+    }
+
+    private void setTweetAction(Bundle savedInstanceState) {
+        if (PrefSystem.isEasyTweetEnabled()) {
+            // EasyTweet
+            if (savedInstanceState == null) {
+                Fragment fragment = new EasyTweetFragment();
+                FragmentTransaction transaction = getSupportFragmentManager().beginTransaction();
+                transaction.replace(R.id.frame_easy_tweet, fragment);
+                transaction.commit();
+            }
+            // Hide tweet button
+            findViewById(R.id.fab).setVisibility(View.GONE);
+        } else {
+            // TweetButton
+            findViewById(R.id.fab).setOnClickListener(v -> {
+                Intent intent = new Intent(v.getContext(), PostActivity.class);
+                if (hasHashTag()) intent.putExtra("TWEET_SUFFIX", mSearchWord);
+                startActivityForResult(intent, 0);
+                overridePendingTransition(R.anim.fade_in_from_bottom, R.anim.none);
+            });
+        }
+    }
+
+    private void setSearchEditActionBar() {
+        // Set custom actionbar
+        ViewGroup root = findViewById(android.R.id.content);
+        View view = getLayoutInflater().inflate(R.layout.edit_search_bar, root, false);
+        getSupportActionBar().setCustomView(view);
+        getSupportActionBar().setDisplayShowCustomEnabled(true);
+
+        // Set edit text
+        mSearchEdit = view.findViewById(R.id.edit_search);
+        mSearchEdit.setOnKeyListener((v, keyCode, event) -> {
+            // When pressed enter
+            if (mSearchEdit.length() != 0 && keyCode == KeyEvent.KEYCODE_ENTER
+                    && event.getAction() == KeyEvent.ACTION_DOWN) {
+                // Close keyboard
+                KeyboardUtils.closeKeyboard(v);
+
+                // Intent to SearchActivity
+                ClickUseCase.searchWord(mSearchEdit.getText().toString());
+                return true;
+            }
+            return false;
+        });
+    }
+
+    private void showAddDialog() {
+        if (TweetHubApp.getMyAccount().getColumns().size() >= 8) {
+            ToastUtils.showShortToast("これ以上追加できません。");
+            return;
+        }
+        DialogUtils.showConfirm(
+                "カラムに追加しますか？",
+                (dialog, which) -> {
+                    // Add column
+                    ColumnArray<Column> columns = TweetHubApp.getMyAccount().getColumns();
+                    boolean isSucceeded = columns.add(
+                            new Column(mSearchWord.hashCode(), mSearchWord, SEARCH_SINGLE, false));
+                    // Check
+                    if (!isSucceeded) {
+                        return;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Replace this if-then-else statement by a single return statement. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://192.168.11.19:9000/coding_rules#rule_key=squid%3AS1126)